packages/vue/src/plugins/index.ts

```diff
@@ -2,3 +2,4 @@ export * from "./test";
 export * from "./translation";
 export * from "./validation";
 export * from "./error";
+export * from "./format";
```

Comment:
```suggestion
export { FormatPlugin } from "./format";
```

Lika bra att börja direkt så det slipper bli en skuld senare.